examples/lock-app/efr32/args.gni

```diff
@@ -22,6 +22,7 @@ declare_args() {
   chip_enable_ota_requestor = true
 }
 
-chip_enable_openthread = true
 pw_log_BACKEND = "${chip_root}/src/lib/support/pw_log_chip"
 pw_assert_BACKEND = "$dir_pw_assert_log"
+chip_enable_openthread = true
+chip_system_config_use_open_thread_udp = true
```

Comment:
Should stay the same as before. `chip_system_config_use_open_thread_udp` got replace by something else and is moved to all efr32 platform and not just example specific

Reply:
chip_enable_openthread = true needs to stay but it was placed a few lines up before. No reason to move it.